pkg/docker-engine/Dockerfile

```diff
@@ -96,7 +96,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends apt-utils bash
 RUN <<EOT
   set -e
   case "$PKG_RELEASE" in
-    ubuntu2004|ubuntu2204)
+    ubuntu2004|ubuntu2204|ubuntu2210)
       if [ "$(dpkg-divert --truename /usr/bin/man)" = "/usr/bin/man.REAL" ]; then
```

Comment:
I guess the only exception now is 18.04? Anything newer than that is now shipping without man-pages as default.

Wondering if we should make a reverse rule (less maintenance), and add a TODO to remove it once 18.04 is EOL, which looks to be April next year; https://github.com/docker/docker-ce-packaging/blob/396301be059a83162d39f5cc569998e7656492ce/Jenkinsfile#L16

Reply:
Ah right that makes sense